src/EA.Weee.Web/Areas/Admin/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -98,10 +104,94 @@ public async Task<ActionResult> ViewEvidenceNoteTransfer(Guid evidenceNoteId)
                 var model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(result.TransferredOrganisationData.Id,
                    result, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]));
 
+                model.CanVoid = IsInternalAdminUser();
+
+                return View(model);
+            }
+        }
+
+        [HttpGet]
+        [AuthorizeInternalClaims(Claims.InternalAdmin)]
+        public async Task<ActionResult> VoidTransferNote(Guid transferEvidenceNoteId)
+        {
+            SetBreadcrumb(BreadCrumbConstant.ManageEvidenceNotesAdmin);
+
+            using (var client = apiClient())
+            {
+                var request = new GetEvidenceNoteTransfersForInternalUserRequest(transferEvidenceNoteId);
+
+                var transferNoteData = await client.SendAsync(User.GetAccessToken(), request);
+
+                if (transferNoteData.Type == NoteType.Transfer && transferNoteData.Status == NoteStatus.Approved)
+                {
+                    var model = mapper.Map<ViewTransferNoteViewModel>(new ViewTransferNoteViewModelMapTransfer(transferNoteData.TransferredOrganisationData.Id,
+                       transferNoteData, TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]));
```

Comment:
the line passing TempData[ViewDataConstant.TransferEvidenceNoteDisplayNotification]you can just pass null to explicitly say that a notification shouldnt be displayed.